.travis.yml

```diff
@@ -1,11 +1,23 @@
-sudo: false
-dist: trusty
```

Comment:
There is a new distribution: "Xenial" based on Ubunto 16.04.
Will that solve the problem of PyGobject dependency?